_build_cfg.yml

```diff
@@ -34,7 +34,7 @@ Distros: openshift-enterprise,openshift-dedicated
 Topics:
   - Name: Overview
     File: index
-  - Name: OpenShift v2 vs. OpenShift v3
+  - Name: Comparing OpenShift Enterprise 2 and OpenShift Enterprise 3
```

Comment:
I moved it to the bottom of the rel notes section. It's in the rel notes section because this is the information from the What's New section leftover from previously. I don't think Architecture is a better fit, as it's not so much about the architecture itself, but rather what's changed.